### DISCUSSION TITLE
Does Trino support Json_Match in Pinot?

### DISCUSSION THREAD

**Discussion by v-jacai** [trinodb/trino#22528] [Q&A]
In order to query index json in pinot, I need to use json_match, it works fine in Pinot, but it's failed after it connects to Trino. the json_extract_scalar works fine, but it don't go thru the index. It will pull everything which is slow.

**Comment by mosabua** [comment]
Functions are typically run in Trino .. and not pushed down. Only a few are pushed down in the Pinot connector https://trino.io/docs/current/connector/pinot.html#pushdown

TO use the native json_match from Pinot the connector would need to support query pass through as table function. That is currently not the case.
